web/html/src/vendors/npm.licenses.txt

```diff
@@ -668,27 +668,7 @@ react-lifecycles-compat v3.0.4
 reactjs/react-lifecycles-compat
 --------------------------------------------------------------------------------
 
-MIT License
-
-Copyright (c) 2013-present, Facebook, Inc.
-
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in all
-copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
```

Comment:
Why are we removing the license text?

Reply:
fixed, thanks. 

It seems it was a bad idea to clean ".md" files with the autoclean option. Some few libraries define their license in a "LICENSE.md" instead of the common "LICENSE". https://github.com/reactjs/react-lifecycles-compat 

I will open a PR on the yarn package to suggest them in removing the "*.md" pattern from the default file